source-code/plugins/i18next/README.md

```diff
@@ -29,32 +29,62 @@ The plugin offers further configuration options that can be passed as arguments.
 
 ```typescript
 type PluginSettings = {
-	pathPattern: string
+	pathPattern: string | { [key: string]: string }
+	variableReferencePattern?: [string] | [string, string]
+	ignore?: [string]
 }
 ```
 
 ### `pathPattern`
 
 To use our plugin, you need to provide a path to the directory where your language-specific files are stored. Use the dynamic path syntax `{language}` to specify the language name. Note that subfile structures are not supported.
 
-**Most common for i18next with namespace support:**
+#### Without namespaces
 
 ```typescript
-pathPattern: "./resources/{language}/*.json"
+pathPattern: "./resources/{language}.json"
 ```
 
-**Only one namespace:**
+#### With namespaces
+
+> Does not get created by 'npx @inlang/cli config init'
 
 ```typescript
-pathPattern: "./resources/{language}/translation.json"
+pathPattern: {
+	common: "./resources/{language}/common.json"
+	vital: "./resources/{language}/vital.json"
+}
 ```
 
-**Language as file name:**
+`key` (prefix): is prefixing the key with a colon
+`values` (path): is the path to the namespace resources
+
+### `variableReferencePattern`
+
+Defines the pattern for variable references. The default is how i18next suggests the usage of placeholders.
+
+default:
 
 ```typescript
-pathPattern: "./resources/{language}.json"
+variableReferencePattern: ["{{", "}}"]
+```
+
+### `ignore`
+
+When you want to ignore files like `languages.json` that are on the same level as your language files, you can ignore them to not brake the `getLanguages` function.
+
+```typescript
+ignore: ["languages.json"]
 ```
 
+## IDE-extension usage
+
+The plugin automatically informs the [IDE extension](https://inlang.com/documentation/apps/ide-extension) how to extract keys and namespaces from your code in order to display inline annotations.
+
+### In-code usage
+
+To learn about namespaces and how to use translation functions in your code, you can refer to [i18next documentation](https://www.i18next.com/principles/namespaces). The plugin is capable of parsing the code and providing the IDE-extension with this information.
+
 ## Contributing
 
```

Comment:
Add how to contribute / join discord or delete contribution headline.

Reply:
It describes how to contribute to code. Have you seen that? I can add the join Community section in the end. @felixhaeberle